static/app/utils/formatters.tsx

```diff
@@ -194,35 +210,45 @@ export function getExactDuration(seconds: number, abbreviation: boolean = false)
       };
     };
 
-    if (value >= WEEK) {
+    if (value >= WEEK || (value && minSuffix === ' weeks')) {
```

Comment:
Can these if statements check agains `precision` instead of `minSuffix` to retain typing and not have the extra space at the beginning?

Reply:
cannot unfortunately since we're comparing to `suffix` within each `if` statement - which is also prefixed with `' '`